js/index.d.ts

```diff
@@ -1,47 +1,159 @@
-/*
-  Typings for @sentry/cli
-*/
+/**
+ * Typings for @sentry/cli
+ */
 declare module '@sentry/cli' {
   export interface SentryCliOptions {
+    /**
+     * Sentry instance
+     */
     url?: string;
+    /**
+     * Authentication token for API, interchangeable with `apiKey`
+     */
     authToken?: string;
+    /**
+     * Authentication token for API, interchangeable with `authToken`
```

Comment:
Is `apiKey` deprecated?

Reply:
I think so, although it's still fully functional. It's a cascade of `SENTRY_AUTH_TOKEN > SENTRY_API_KEY > config.token > config.api_key`

https://github.com/getsentry/sentry-cli/blob/758d6ffb324691f54223098403f1d35684584dc0/src/config.rs#L542-L554